lib/compare_with_wikidata.rb

```diff
@@ -142,6 +142,8 @@ def csv_from_url(file_or_url)
       CSV.parse(RestClient.get(file_or_url).to_s, headers: true, header_converters: :symbol, converters: nil).map(&:to_h)
     rescue CSV::MalformedCSVError
       raise MalformedCSVError.new("The URL #{file_or_url} couldn't be parsed as CSV. Is it really a valid CSV file?")
+    rescue RestClient::Exception => e
+      raise Exception.new("There was an error fetching: #{file_or_url} - the error was: #{e.message}")
```

Comment:
I think this should probably use a custom subclass of `StandardError`, like you've done with `MalformedCSVError ` above, rather than raising Ruby's built-in `Exception` class.

See [the `Exception` docs](http://ruby-doc.org/core-2.4.1/Exception.html) for more details.